README.md

```diff
@@ -485,6 +488,25 @@ The converted DAG uses the `BashOperator` in Airflow.
 
 Not all FS operations are currently idempotent. This will be fixed.
 
+## Java Example
+
+The Java example can be run as:
+
+`o2a -i examples/java -o output/java`
+
+Make sure to first copy `examples/fs/configuration.template.properties`, rename it as
+`configuration.properties` and fill in with configuration data.
+
+### Output
+In this example the output will appear in `/output/java/test_java_dag.py`.
```

Comment:
Using the `o2a ...` command the output will actually be: `/output/java/java.py`

Reply:
I copied it from other operators. I will create a small PR separately to fix all of them in a moment.

Reply:
Resolved in #307

Reply:
Added a comment there